environments/sbox/sbox.tfvars

```diff
@@ -27,6 +27,7 @@ frontends = [
     custom_domain    = "idam-web-public.sandbox.platform.hmcts.net"
     backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
     certificate_name = "STAR-sandbox-platform-hmcts-net"
+    ip_whitelist     = ["85.92.217.166"]
```

Comment:
have you tested this already in sandbox?

otherwise I would suggest uncommenting the sandbox global apply and making sure that it does what you want first before we mainline support for this

Reply:
Asserted in sandbox.